extension/src/format/sexpression.ts

```diff
@@ -1134,31 +1180,36 @@ export class Sexpression extends LispAtom {
 
     formatting(startColumn: number, linefeed?: string): string {
 
-        gMaxLineChars = maximumLineChars();
-        if (gMaxLineChars < 60)
-            gMaxLineChars = 60;
+        if (!gHasMaxLineChars) {
```

Comment:
I don't know if this change is correct or not, it seems that when running this method in product, the flag is false, and then the original code that Hualin wrote won't run? is my understanding correct?
Anyway, I suggest not to change feature logic like this only for adding test cases. 
If it's really hard to test against sth. without changing the logic, we can do it via. E2E.